src/components/MainHeader.vue

```diff
@@ -235,9 +244,9 @@ export default {
   }
   .main-nav {
     position: fixed;
-    top: $base-line-height;
+    top: $base-line-height * 0.75;
```

Comment:
So, from my testing, removing this `top` CSS property, fixes the styling issue. This is because the code below (on line 369) takes into effect because of the mobile breakpoint.

Reply:
Remove that causes this:
![image](https://user-images.githubusercontent.com/5607236/30754437-91d31eac-9fba-11e7-8b3e-552448e8fbbd.png)

Reply:
Can you get me a screenshot of the issue you're seeing?

Since this PR fixes some serious bugs with what's on dev, could make the hamburger location a follow up issue and get this merged if everything else is ok?

Reply:
Hey Wes, here is what I'm seeing. This is on Chrome and iOS Simulator. Both have the viewport of the iPhone 5.

<img width="322" alt="screen shot 2017-09-26 at 7 06 14 pm" src="https://user-images.githubusercontent.com/2737742/30888410-642487ca-a2ee-11e7-9840-e6ca3e486bd1.png">
<img width="320" alt="screen shot 2017-09-26 at 7 06 45 pm" src="https://user-images.githubusercontent.com/2737742/30888411-642654f6-a2ee-11e7-8784-5775e58a4e6d.png">